plugins/attacks.py

```diff
@@ -19,8 +19,8 @@ def is_self(conn, target):
     """ Checks if a string is "****self" or contains conn.name. """
     if re.search("(^..?.?.?self|{})".format(re.escape(conn.nick)), target, re.I):
         return True
-    else:
-        return False
+
```

Comment:
Surely this could be replaced with something like `return  re.search("(^..?.?.?self|{})".format(re.escape(conn.nick)), target, re.I) is not None`